packages/main/src/ListItem.js

```diff
@@ -354,12 +373,13 @@ class ListItem extends ListItemBase {
 
 	get _accInfo() {
 		return {
-			role: this.role,
+			role: this.accessibleRole || this.role,
 			ariaExpanded: undefined,
 			ariaLevel: undefined,
 			ariaLabel: ListItem.i18nBundle.getText(ARIA_LABEL_LIST_ITEM_CHECKBOX),
 			ariaLabelRadioButton: ListItem.i18nBundle.getText(ARIA_LABEL_LIST_ITEM_RADIO_BUTTON),
 			ariaSelectedText: this.ariaSelectedText,
+			ariaHaspopup: this._ariaHasPopup || undefined,
```

Comment:
Yes, we need it! _ariaHasPopup is of type String, and the default value is "" (empty string). Leaving it (without set to undefined) will result to existing aria-haspopup attribute with no value, instead of missing aria-haspopup attribute.